src/components/assessment/assessmentShape.ts

```diff
@@ -164,8 +164,11 @@ type ExternalLibrary = {
   symbols: string[];
 };
 
+export type Variant = 'lazy' | 'non-det' | 'default';
```

Comment:
@arsalanc-v2 why do we need to export this again, if it's defined in js-slang?

Reply:
Yeah it's not needed